Add "Network interfaces should not have public IPs" - Corp - ADO WIT 25739 (#1170)
Azure/Enterprise-Scale: docs/wiki/ALZ-Policies.md

| **Enforces the use of Premium Databricks workspaces**          | **Deny non-premium Databricks sku**                            | `Policy Definition`, **Custom**     | Enforces the use of Premium Databricks workspaces to make sure appropriate security features are available including Databricks Access Controls, Credential Passthrough and SCIM provisioning for AAD. | Deny              | 1.0.0   |
| **Enforces the use of vnet injection for Databricks**          | **Deny Databricks workspaces without Vnet injection**          | `Policy Definition`, **Custom**     | Enforces the use of vnet injection for Databricks workspaces.                                                                                                                                          | Deny              | 1.0.0   |

> | **Deny the the network interfaces which are configured with any public IP**          | **Network interfaces should not have public IPs**          | `Policy Definition`, **Built-in**     | This policy denies the network interfaces which are configured with any public IP under the assigned scope.                                                                                                                                          | Deny              | 1.0.0   |

Reviewer: Typo here. "the the"

Author: committed your suggestion for partial replacement below...

> | **Deny the the network interfaces which are configured with any public IP**          | **Network interfaces should not have public IPs**          | `Policy Definition`, **Built-in**     | This policy denies the network interfaces which are configured with any public IP under the assigned scope.                                                                                                                                          | Deny              | 1.0.0   |

Reviewer: ```suggestion
| **Deny network interfaces having a public IP associated**          | **Network interfaces should not have public IPs**          | `Policy Definition`, **Built-in**     | This policy denies network interfaces from having a public IP associated to it under the assigned scope.                                                                                                                                          | Deny              | 1.0.0   |
```
